packages/trpc/server/routers/viewer/apps.tsx

```diff
@@ -255,9 +255,7 @@ export const appsRouter = router({
       })
     )
     .mutation(async ({ ctx, input }) => {
-      let appKey = deriveAppDictKeyFromType(input.type, appKeysSchemas);
-      if (!appKey) appKey = deriveAppDictKeyFromType(input.slug, appKeysSchemas);
-      const keysSchema = appKeysSchemas[appKey as keyof typeof appKeysSchemas];
+      const keysSchema = appKeysSchemas[input.slug as keyof typeof appKeysSchemas];
```

Comment:
We are already using dirName for this https://github.com/calcom/cal.com/blob/55a4510167db026686768dec4058ff971a9d8747/packages/trpc/server/routers/viewer/apps.tsx#L66. So made it consistent.

It was causing keys to be not saved for any new app